src/pages/AddUnreportedExpense.tsx

```diff
@@ -78,7 +79,7 @@ function AddUnreportedExpense({route}: AddUnreportedExpensePageType) {
     });
 
     const fetchMoreUnreportedTransactions = () => {
-        if (!hasMoreUnreportedTransactionsResults || isLoadingUnreportedTransactions) {
+        if (!hasMoreUnreportedTransactionsResults || !!isLoadingUnreportedTransactions || shouldShowUnreportedTransactionsSkeletons) {
```

Comment:
Why do we need `shouldShowUnreportedTransactionsSkeletons` here as I think both conditions are already covered with `!hasMoreUnreportedTransactionsResults || !!isLoadingUnreportedTransactions`

Reply:
for some reason it prevented sending a new request when skeletons were rendered, bc the previous condition didn't work. Now I see that it doesn't work anymore, so I need to find another solution

Reply:
@Pujan92 restored this old one